packages/notion/src/schema/index.ts

```diff
@@ -15,7 +16,7 @@ type DataTypesObject = {
 }
 
 type DatabaseInfo = {
-  [key in DatabaseType]: {
+  [key in DatabaseType]?: {
```

Comment:
Small step into making this more configurable by being able to not have every `DatabaseType` required.